StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -5,8 +5,44 @@
 
 import Foundation
 
-/// An `Event` object represent an event in the chat system.
+/// An `Event` object representing an event in the chat system.
 public protocol Event {
     /// The underlying raw type of the incoming string.
     static var eventRawType: String { get }
 }
+
+/// The DTO object mirroring the JSON representation of an event.
+struct EventPayload<ExtraData: ExtraDataTypes>: Decodable {
```

Comment:
It looks like should be `ChannelEventPayload`.

Reply:
Currently, it contains only channel data but it will contain all possible data for all events. This is **just** the struct which serialize the event JSON payload. No logic here.